STEP_Project/OUListContent.js

```diff
@@ -7,6 +7,27 @@ function addRemoveButtonClick (event)
     if (event.target.getAttribute("class")!=='bClass') return;
 }
 
+/**
+ * Adds a button to every visible row when the page is first loaded.
+ * @param {HTML Object} tabl - contains OU rows
+ */
+function initButtons(tabl)
+{
+    let OUs = tabl.rows;
+    let numOUs = OUs.length;
+    let numRows = numOUs-1;
+    addButtons(OUs,numRows);
+
+    /**
+     * This is to handle an edge case:
+     * It deals with the last row in the table that doesn't actually correspond to an OU
+     */
```

Comment:
This should be a comment and not doc. Checkout
https://stackoverflow.com/questions/3607641/javadoc-comments-vs-block-comments#:~:text=*%2F%20.,files%20using%20the%20javadoc%20tool.&text=Doc%20comments%20are%20meant%20to,the%20source%20code%20at%20hand..

Reply:
Changed it to a comment